pkg/arvo/sys/vane/jael.hoon

```diff
@@ -308,11 +308,15 @@
       ::
       ::  start subscriptions
       ::
-      ::  get everything from azimuth-tracker because jael subscriptions
-      ::  seem to be flaky for now
-      ::
       =.  +>.$  (poke-watch hen %azimuth-tracker nod.own.pki)
       =.  +>.$
+        ::  get everything from azimuth-tracker because jael subscriptions
+        ::  seem to be flaky for now
+        ::
+        ?:  &
```

Comment:
I feel like such things should be marked with an `XX` also, for that "temporary and should be fixed" feel.